bears/python/PyUnusedCodeBear.py

```diff
@@ -15,16 +15,16 @@ class PyUnusedCodeBear(LocalBear):
     CAN_DETECT = {'Unused Code'}
 
     def run(self, filename, file,
-            remove_all_unused_imports: bool=False,
+            remove_all_unused_imports: bool=True,
             remove_unused_variables: bool=True):
         """
         Detects unused code. By default this functionality is limited to:
 
         - Unneeded pass statements.
-        - Unneeded builtin imports.
+        - Unneeded imports.
 
         :param remove_all_unused_imports:
-            True removes all unused imports - might have side effects
+            False removes all unused builtin imports - might have side effects
```

Comment:
Yes it does. https://github.com/coala/coala-bears/issues/1824. The output snippet by zimmermann shows it had removed rstcheck as well